config.sh

```diff
@@ -10,6 +10,7 @@ OPENBLAS_VERSION=0.2.18
 source gfortran-install/gfortran_utils.sh
 
 function build_wheel {
+  export FFLAGS="$FFLAGS -fPIC -Wl,-strip-all"
```

Comment:
Is `-fPIC` right on 32-bit platforms?

Reply:
Also, adding `-Wl,-strip-all` seems to be included in multibuild https://github.com/matthew-brett/multibuild/commit/b2748e5d4d2ae6e45b130bdf8e24361083341e9a --- maybe we can bump the version to that commit instead and not set FFLAG here? Or maybe I'm missing something?

Reply:
`-fPIC` should work on all platform as far as I know.

That commit in `multibuild` only sets `-Wl,-strip-all` if no other `FFLAGS` are set, i.e. it supports that you can override the compile options if you don't want your things stripped. Thus we need to supply all flags here.